src/extension.ts

```diff
@@ -1,15 +1,20 @@
 "use strict";
 
-import * as vscode from "vscode";
 import * as fs from "fs";
 import path = require("path");
-import { getSnippet } from "./snippet";
+import * as vscode from "vscode";
 import { config } from "./config";
-import { showComments, lineChangeFlag } from "./showComments";
-import showCommentListPanel from "./webView/webViewPanel";
+import { languages } from 'vscode';

```

Comment:
Since we are already importing * from vscode, is a separate import for languages required here?